cibuildwheel/macos.py

```diff
@@ -47,31 +47,41 @@ def call(args, env=None, cwd=None, shell=False):
 
         # if this version of python isn't installed, get it from python.org and install
         python_package_identifier = 'org.python.Python.PythonFramework-%s' % config.version
+        installation_bin_path = '/Library/Frameworks/Python.framework/Versions/{}/bin'.format(config.version)
         if python_package_identifier not in installed_system_packages:
             # download the pkg
             call(['curl', '-L', '-o', '/tmp/Python.pkg', config.url])
             # install
             call(['sudo', 'installer', '-pkg', '/tmp/Python.pkg', '-target', '/'])
 
+        # Python bin folders on Mac don't symlink python3 to python, so we do that
+        # so `python` and `pip` always point to the active configuration.
+        if os.path.exists('/tmp/cibw_bin'):
```

Comment:
Is it worth turning this little `if exists: rmtree; mkdirs` snippet into a function? (Is it used on other operating systems as well?)

Reply:
It's used on windows as well, but I don't mind repeating it, since it's clear and concise enough I think